tasks/buildsources.js

```diff
@@ -107,6 +107,15 @@ module.exports = function(grunt) {
 				config.rawSource = '\n// '+featureName + '\n' + config.rawSource;
 
 				var featurePath = path.join(destFolder, featureName+'.json');
+                var featureDir = path.dirname(featurePath);
+                mkdir(featureDir, function(err) {
```

Comment:
Could you change this to be a synchronous action instead of asynchronous? The reason being that this introduces a race condition where the directory creation happens after the file creation on line 199 in this file and an error would be thrown.